modules/nlb/target/main.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_lb_target_group" "target_group" {
```

Comment:
Do we want to specify a `deregistration_delay` value here? Or are we ok with the default one which is 300 seconds?

Reply:
We don't set this in our production SaaS, so I assume the default is probably fine. During `draining` status, new connections can't go there, so it is unlikely to matter.